playbooks/gophercloud-acceptance-test/run.yaml

```diff
@@ -7,6 +7,7 @@
         - 'manila'
         - 'designate'
         - 'zun'
+        - 'lbaas'
```

Comment:
You should add load balancer accpetance tests in 52 line, currently only part of test have been enabled. Other part of code looks good to me.

Reply:
Done

Reply:
Thank you.